Dockerfile.aarch64

```diff
@@ -1,31 +1,39 @@
-FROM lsiobase/alpine.python.arm64:3.8
+FROM lsiobase/alpine.python:3.9
```

Comment:
unfortunately our baseimages are still not multi-arch so we need to keep those as `lsiobase/alpine.python.arm64:3.9`

Reply:
fixed in later commit